test/fixtures/profiles/raise_outside_control/controls/raises.rb

```diff
@@ -0,0 +1,11 @@
+# Ref https://github.com/inspec/inspec/issues/5107
+# Users may add code outside controls without exception handling.
+# While this is a user mistake, users may not know about it if we don't report it...
+
+raise StandardError, 'Something unforeseen...'
+
```

Comment:
Would this feature also work for a `raise` inside of a control block but outside of a `describe` block? I feel like that code also gets evaluated but it also might be a case you're not looking to solve with this issue.

Reply:
See the issue: https://github.com/inspec/inspec/issues/5107#issuecomment-648726990

There's at least one other case that needs a different fix, it will be filed as a separate issue.